### PR TITLE
Updating single user db

### PR DESCRIPTION
**Pull request by garv5014**
Updated the database to have a is single user field. 

This request needs to fulfill these reqs.

- Record in db if they are single user
  - Make the person a single user with a flag in the database on the customer. 
- UI distinctions (single user shouldn't see patient selector, welcome card on dashboard greets them)
  - Use the boolean in the database to dynamically show that. 
- Refactor customer service into tanstack hooks
  - Doesn't have tanstack hooks currently so integrate it with tanstack hooks

I didn't move it but it will hide if the user isn't a single use we can move this functionality into a new story.
- move patient selector onto its own nested layout nav instead of being on the main navbar 
  - Move it so that it is out of the global context on the top navbar to a different context.

### CODE REVIEW

File: .vscode/launch.json
Comment (Cwighty) [inline .vscode/launch.json:1]: Is this why the C# plugin stopped working?

**Review by Cwighty — APPROVED** [review]
Looks good

File: .vscode/launch.json
Comment (garv5014) [reply .vscode/launch.json:1]: Oo maybe I dont temebrt adding that

Comment (Cwighty) [reply .vscode/launch.json:1]: I meant did this fix it lol

Comment (Cwighty) [reply .vscode/launch.json:1]: I just never thought to change that when I was updating

**Pull request merged**